libs/clients/syslumenn/src/lib/syslumennClient.utils.ts

```diff
@@ -479,7 +480,7 @@ export const mapEstateRegistrant = (
     marriageSettlement: syslaData.kaupmaili ?? false,
     office: syslaData.embaetti ?? '',
     caseNumber: syslaData.malsnumer ?? '',
-    dateOfDeath: syslaData.danardagur ?? '',
+    dateOfDeath: syslaData.danardagur ?? new Date(), // Todo: suitable backup for date
```

Comment:
Is this todo something you'd like to consider addressing?

Reply:
Decided to remove and keep this as the fallback.